README.md

```diff
@@ -88,7 +88,7 @@ Action::actWhen($isChuckNorrisMighty, new ChuckNorris($data));
 Action::actUnless($isChuckNorrisPuny, new ChuckNorris($data));
 ```
 
-The usage is nearly identical to calling the methods directly on the action as mentioned in the section above. The benefit here is that you can easily test actions using `Action::shouldReceive('act')` or whichever method you used!
+The usage is nearly identical to calling the methods directly on the action as mentioned in the section above. The benefit here is that you can easily test actions using `Action::shouldReceive('act')`, `Action::shouldReceive('actWhen')` or `Action::shouldReceive('actUnless')`.
```

Comment:
I think if I understood correctly here, we can display the 3 options without taking much more space.